srcpkgs/YACReader/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'YACReader'
+pkgname=YACReader
+version=9.5.0
+revision=1
+wrksrc=yacreader-${version}
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-script-devel qt5-declarative-devel
+ qt5-quickcontrols qt5-multimedia-devel qt5-imageformats poppler-qt5-devel
+ qrencode-devel glu-devel libunarr-devel"
```

Comment:
`qrencode-devel` and `glu-devel` are not needed in `hostmakedepends`

Reply:
Thanks! How do you find that out?

Reply:
Try and error. Start with no libraries in hostmakedepends and add only what is necessary.